docs/source/examples.rst

```diff
@@ -16,13 +16,13 @@ PBS Deployments
    cluster = PBSCluster(queue='regular',
                         project='DaskOnPBS',
                         local_directory='$TMPDIR',
-                        threads=4,
+                        cores=24,
                         processes=6,
                         memory='16GB',
                         resource_spec='select=1:ncpus=24:mem=100GB')
 
-   cluster = PBSCluster(processes=18,
-                        threads=4,
+   cluster = PBSCluster(cores=24,
```

Comment:
The direct translation of the previous snippet would be `cores=72,processes=18`, although possible this seems a bit over the top so I switched to more reasonable numbers.